kahuna/public/js/components/gr-image-usage/gr-image-usage-list.html

```diff
@@ -0,0 +1,22 @@
+<div class="image-info__group">
+    <span class="flex-container gr-image-usage__field_title">
+        {{ctrl.type}} ({{ctrl.usages.length}})
+    </span>
+
+    <ul>
+        <li class="gr-image-usage__record" ng-repeat="data in ctrl.usages.data">
+            {{data.title}}
+            <div class="more-info">
+                <span class="date-added">{{ctrl.formatTimestamp(data.dateAdded)}}</span>
```

Comment:
This function on the controller would run the same number of times though...?

In both cases, if the expression is immutable, you can/should make it a one-time binding by prefixing with `::`, e.g. `{{:: ctrl.formatTimestamp(data.dateAdded)}}`, which prevents the creation of a watcher for it.